internal/cmd/asyncapi/command_import.go

```diff
@@ -414,14 +436,14 @@ func addTopicTags(details *accountDetails, subscribe Operation, topicName string
 		tagConfigs = append(tagConfigs, srsdk.Tag{
 			TypeName:   tag.Name,
 			EntityType: "kafka_topic",
-			EntityName: details.srCluster.Id + ":" + details.clusterId + ":" + topicName,
+			EntityName: fmt.Sprintf("%s:%s:%s", details.srCluster.Id, details.clusterId, topicName),
 		})
 	}
 	err := addTagsUtil(details, tagDefConfigs, tagConfigs)
 	if err != nil {
 		return err
 	}
-	output.Printf("Tags added to Kafka topic %s\n", topicName)
+	output.Printf("Tags added to Kafka topic \"%s\"\n", topicName)
```

Comment:
I don't see this output being tested, can you include this (and any other missing output) in your integration tests?